events/code_review_events/workflow.py

```diff
@@ -216,9 +257,140 @@ def publish_results(self, payload):
 
         return True
 
-    def parse_pulse(self, payload):
+    async def got_bugbug_test_select_end(self, payload):
+        bugbug_task_id = payload["status"]["taskId"]
+
+        # Get the Phabricator diff ID from bugbug task definition.
+        bugbug_task = self.community_tc["queue"].task(bugbug_task_id)
+        diff_id = int(bugbug_task["extra"]["phabricator-diff-id"])
+
+        # If this diff does not belong to a revision we pushed to try, return.
+        if diff_id not in self.diff_to_push:
+            logger.warning(
+                "bugbug test select notification for a revision we did not push to try",
+                payload=payload,
+            )
+            return
+
+        # Retrieve artifacts from bugbug test selection task.
+        try:
+            failure_risk = self.community_tc["queue"].getLatestArtifact(
+                bugbug_task_id, "public/failure_risk"
+            )
+            assert isinstance(failure_risk, int)
+        except taskcluster.exceptions.TaskclusterRestFailure as e:
+            if e.status_code == 404:
+                logger.error(
+                    f"Missing failure_risk artifact in bugbug task", task=bugbug_task_id
+                )
+                return
+            raise
+
+        # If the failure risk is low, don't trigger tests.
+        if failure_risk == 0:
```

Comment:
The artifact can simply contain `0` ? :rofl:

Reply:
Either 1 for risky or 0 for not risky